src/base/BasePaymaster.sol

```diff
@@ -55,7 +56,7 @@ abstract contract BasePaymaster is Ownable {
      * Unlock the stake, in order to withdraw it.
      * The paymaster can't serve requests once unlocked, until it calls addStake again
      */
-    function unlockStake() external onlyOwner {
+    function unlockStake() external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
I would also make this admin/manager and restrict the `withdrawStake` to admin only